scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,20 @@
+-----------------------------------
+-- Area: Oldton Movalpolos
+--  MOB: Goblin Wolfman
+-----------------------------------
+require("scripts/zones/Oldton_Movalpolos/MobIDs");
+require("scripts/globals/settings");
+require("scripts/globals/status");
+-----------------------------------
+
+function onMobInitialize(mob)
+mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 55);
```

Comment:
gotta indent
```lua
stuff
    things inside stuff
end
```
4 spaces per level of indention.

```lua
function example(argumnet1, argument2)
    if conditions then
        if otherConditions then
            doThings
        end
    end
end
```

and make sure the value for that mobmod is retail accurate. I don't thin 55 seconds is correct. its usually 3 or 5 minutes